src/main/app.ts

```diff
@@ -380,6 +380,12 @@ export class KandoApp {
 
         // On all platforms except Windows, we show the window after we moved it.
         if (process.platform !== 'win32') {
+          // Ensure the window is on the current worksspace before showing
+          this.window.setVisibleOnAllWorkspaces(true, { skipTransformProcessType: true });
+          setTimeout(() => {
+          this.window.setVisibleOnAllWorkspaces(false, { skipTransformProcessType: true });
+          }, 100);
```

Comment:
As this does not seem to be required on Linux, I would prefer if that was only done on macOS. So maybe move this before the `if (process.platform !== 'win32') {` into an `if (process.platform === 'darwin' {`?

Also, you could mention #461 in the comment so that we know in the future where this weirdo comes from :smile:

Reply:
Done. Pushed a new commit with the change 👍️